docs/samples/drift-detection/alibi-detect/cifar10/requirements_notebook.txt

```diff
@@ -1,3 +1,7 @@
 alibi-detect>=0.4.0
+httplib2>=0.20.2
```

Comment:
And old version of httplib2 is resolved as an indirect dependency and that causes issues. Hence, I added a direct dependency to a working version.